src/frontends/paddle/src/op/dequantize_linear.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs dequantize_linear(const NodeContext& node) {
+    // extract the INPUTS
+    const auto x = node.get_input("X");
+    const auto scale = node.get_input("Scale");  // type: float or 1-D
+    const auto zero_point = node.get_input("ZeroPoint");
+
+    // assert shape of scale and zero_point
+    PADDLE_OP_CHECK(node, scale.get_partial_shape().rank().is_static(), "dequantize: scale rank must be static!");
+    PADDLE_OP_CHECK(node,
+                    zero_point.get_partial_shape().rank().is_static(),
+                    "dequantize: zero_point rank must be static!");
+    const auto& scale_shape = scale.get_partial_shape();
+    const auto& scale_shape_length = scale_shape.rank().get_length();
+
+    if (scale_shape_length == 1) {
+        PADDLE_OP_CHECK(node,
+                        scale.get_partial_shape() == zero_point.get_partial_shape(),
+                        "dequantize_linear shape of scale and zero_point doesn't match.");
+    } else if (scale_shape_length == 2) {
+        PADDLE_OP_CHECK(node,
+                        scale.get_partial_shape()[1] == zero_point.get_partial_shape()[0],
+                        "dequantize_linear shape of scale and zero_point doesn't match.");
+    } else {
+        PADDLE_OP_CHECK(node, false, "dims of scale should not be greater than 2.");
+    }
+
+    const auto bit_length = node.get_attribute<int32_t>("bit_length");
+    const auto range = (1 << (bit_length - 1)) - 1;
+    const auto range_node = std::make_shared<default_opset::Constant>(element::f32, Shape{1}, (1.0 / range));
+    const auto real_scale = std::make_shared<default_opset::Multiply>(scale, range_node);
```

Comment:
Would you please explain this code snippet here? Why "scale" has to be multiplied by the range stuff? What does the input "scale" stand for?

Reply:
> Would you please explain this code snippet here? Why "scale" has to be multiplied by the range stuff? What does the input "scale" stand for?

The PaddleSlim scale value is not equal to scale definition in OpenVINO. 
scale_ov = scale_pdpd / 127.